tests/input_dir/configs/default/config.yaml

```diff
@@ -1,4 +1,5 @@
 tool: defaultrunner
 tool_version: 1.0.0
 phenotype_only: False
```

Comment:
Change this to 
```yaml
variant_analysis: False
gene_analysis: True
disease_analysis: True
```
i.e. 'phenotype_only' is a weird amalgam of gene and variant...